### ISSUE TITLE
bug - make sure you can click on a location on the world map and be taken to the dashboard (nice to have - add latest value to popup)

### ISSUE THREAD

**Issue by jvntf** [openaq/openaq.org#571]

**Comment by jvntf** [comment]
@rwegener2 i cannot replicate any bug here. can you confirm and / or close?

**Comment by rwegener2** [comment]
@jvntf you are right the popup does appear to be working on the frontend.

To rebrand this ticket a bit then I'd say the task would be to make sure that when you click on a mobile square a pop up still shows up (Ex. ozone measurements in southern Austria)

**Comment by rwegener2** [comment]
Actually it looks like some of the mobile measurements are showing pop ups, but the locations themselves are shown as square points.  Maybe wait on this ticket until the styling of the visualization of mobile data is confirmed with @AliceR

**Comment by rwegener2** [comment]
A couple of things I noticed about the popups:
* The source doesn't seem to link to anything.  We could add the source link or for simplicity I would totally support just listing the source as text.
* "View More" is a bit generic - we could change this to "View Location" or "View Dashboard"

**Comment by AliceR** [comment]
These should all be tackled in #579
